SwiftVocabulary/SwiftVocabulary/View Controllers/WordsTableViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  WordsTableViewController.swift
+//  SwiftVocabulary
+//
+//  Created by Trencadi on 5/20/20.
+//  Copyright © 2020 ladonnamayfield. All rights reserved.
+//
+
+import UIKit
+
+class WordsTableViewController: UITableViewController {
+    
+    var vocabWords: [VocabularyWord] = [
+        VocabularyWord(word: "Guard Let", definition: "A guard statement is used to transfer program control out of a scope if one or more conditions are not met. (Swift Programming Language Guide, 5.2)"),
+        VocabularyWord(word: "Optional Chaining", definition: "Optional Chaining is a process for querying and calling properties, methods and subscripts on an optional that might be currently nil. (Swift Programming Language Guide, 5.2)")]
```

Comment:
Nice job with this, I appreciate these being on their own lines and the nice definitions for these terms.